CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+CHANGES IN VERSION 1.8.0 (from 1.7.2)
+===================================
+
+Appium 1.8.0 introduces full support for the [W3C WebDriver specification](https://www.w3.org/TR/webdriver/)
+([digested version of the spec](https://github.com/jlipps/simple-wd-spec)).
+
+#### General
+* Minimum [NodeJS]() version moved to 6
+* Add methods for
+  * getting and setting the clipboard contents
+  * get full screen screenshots
```

Comment:
We've also fixed error codes handling and updated the webdriver error codes: 
https://github.com/appium/appium-base-driver/pull/192
https://github.com/appium/appium-base-driver/pull/193